Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -69,9 +69,12 @@ void VideoConfig::Load(const std::string& ini_file)
   settings->Get("OverlayStats", &bOverlayStats, false);
   settings->Get("OverlayProjStats", &bOverlayProjStats, false);
   settings->Get("DumpTextures", &bDumpTextures, false);
-  settings->Get("HiresTextures", &bHiresTextures, false);
-  settings->Get("ConvertHiresTextures", &bConvertHiresTextures, false);
-  settings->Get("CacheHiresTextures", &bCacheHiresTextures, false);
+  if (!settings->Get("CustomTextures", &bCustomTextures, false))
+  {
+	  settings->Get("HiresTextures", &bCustomTextures, false);
+  }
```

Comment:
@ZexaronS [The reason is that it is part of Dolphin's Coding Style Guidelines](https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md#conditionals).

Also [this](https://stackoverflow.com/questions/14167033/visual-studio-replace-tab-with-4-spaces/14167067) is the first result when you [USE A SEARCH ENGINE OF YOUR CHOICE] "visual studio space tab". Really not that hard.